src/pages/graphql/announcing-our-new-graphql-bootcamp.md

```diff
@@ -25,7 +25,7 @@ GraphQL works differently. It is a typed query language that defines *how *to as
 
 In the end, GraphQL will give you a sleeker codebase that is way easier to maintain, and will help speed up your front-end dev time as a result!
 
-These are just some of the reasons why we’re introducing a new [GraphQL Bootcamp](https://reactgraphql.academy/graphql/training/bootcamp/?utm_medium=direct&utm_source=blog&utm_campaign=graphql_exp&utm_content=graphql_announcement). We want to show more developers how they can benefit from implementing GraphQL and bring beautiful, sleek code to the community.
+These are just some of the reasons why we’re introducing a new [GraphQL Bootcamp](https://reactgraphql.academy/graphql/training/bootcamp/). We want to show more developers how they can benefit from implementing GraphQL and bring beautiful, sleek code to the community.
```

Comment:
@ric9176 internal links should not have the https and domain, otherwise, the Link component will treat them as a different site and won't do the HMTL5 history.pushState. Therefore a round trip to the server to load the new page